modules/installation-custom-aws-vpc.adoc

```diff
@@ -219,7 +258,12 @@ To ensure that the subnets that you provide are suitable, the installation progr
 * All the subnets that you specify exist.
 * You provide private subnets.
 * The subnet CIDRs belong to the machine CIDR that you specified.
+ifndef::aws-outposts[]
 * You provide subnets for each availability zone. Each availability zone contains no more than one public and one private subnet. If you use a private cluster, provide only a private subnet for each availability zone. Otherwise, provide exactly one public and private subnet for each availability zone.
+endif::aws-outposts[]
+ifdef::aws-outposts[]
+* You provide subnets for each availability zone. Each availability zone contains exactly one public and one private subnet in the AWS region (not created in the Outpost instance). The availability zone in which the Outpost instance installed in should include one aditional private subnet in the Outpost instance.
+endif::aws-outposts[]
 * You provide a public subnet for each private subnet availability zone. Machines are not provisioned in availability zones that you do not provide private subnets for.
 
 If you destroy a cluster that uses an existing VPC, the VPC is not deleted. When you remove the {product-title} cluster from a VPC, the `kubernetes.io/cluster/.*: shared` tag is removed from the subnets that it used.
```

Comment:
@nirarg Is the part about the tag true for an outpost cluster, too?

Reply:
The VPC tagging should have the same behavior unrelated if it includes subnet in Outposts or not
But need to validate this